### ISSUE TITLE
Does this still work?

### ISSUE THREAD

**Issue by Marketingguru** [shujahm/Facebook-Scrapper#1]
I cant seem to get this script to work...Graph returned an error: Invalid appsecret_proof provided in the API argument and can you just change the api to 2.9? I want to grab the names and ids of people who did reacted to a post also.

**Comment by shujahm** [comment]
Yes this script is still valid and I often use it.The error is (based on my experience) almost certainly correct; it means you're proving an invalid appsecret_proof with your API call
Assuming you're using the standard PHP SDK without modifications, the most likely reasons for this are:

You configured the wrong app ID in the SDK code
You configured the wrong app secret in the SDK code
You're trying to use an access token from the wrong / another app

**Comment by jelordreygulle** [comment]
Can you please add the instrucstion on running the script Sir. Thanks

**Comment by shujahm** [comment]
@jelordreygulle yes the script still runs, all you have to do is to add the facebook credentials as mentioned in the README.md file and then just call the respect PHP script through browser or CLI

**Comment by jelordreygulle** [comment]
By the way Sir if you dont mind , have you tried creating a custom audience or getting target audience  using the ff. https://developers.facebook.com/docs/marketing-api/buying-api/targeting ,API'S.
